### PR TITLE
Accessibility branch for changes

### PR DESCRIPTION
**Pull request by monika-soam**
**List of all items checked below:-**

- "Page Title - No Issue found"
- "Image text alternatives-No Issue found"
- "Headings- No Issue Found"
- "Color-contrast- No issue found"
- "Resize-No Issue found"
- "Moving content- Added the transitions for buttons"
- "Media- No Issue found"
- Checked the linters
- Followed the GitHub flow

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,241 +1,280 @@
...
+              <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
+              <ul class="badge-list">
                 <li class="badge bg-gray oil ruby-rails-badge">Ruby on rails</li>
                 <li class="badge bg-gray oil css-badge">css</li>
                 <li class="badge bg-gray oil js-badge">JavaScript</li>
                 <li class="badge bg-gray oil html-badge">html</li>
-            </ul>
-                <button class="btn-green btn-projects" type="button">See Project</button>
+              </ul>
+              <button class="btn-green btn-projects" type="button">See Project</button>
```
Comment (keroloussamy) [inline index.html:100]: - [ ] Kindly change your button color to pass the color contrast it, you can use this [website](https://www.aditus.io/) to check it until you pass it.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @monika-soam, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (monika-soam) [reply index.html:100]: @keroloussamy. This is not my code I have only done pair proh=gramming on this code. I don't understand how this comes to you and would you mind giving me an example of how to apply this?

Comment (monika-soam) [reply index.html:100]: @I am trying to put the link and checking but unable to understand it properly as it is not my code. would you mind me letting know the proper way to understand if I am doing some mistakes?

Comment (monika-soam) [reply index.html:100]: @keroloussamy I am trying to put the link and checking but unable to understand it properly as it is not my code. would you mind me letting know the proper way to understand if I am doing some mistakes? I really didn't get it.

Comment (keroloussamy) [reply index.html:100]: @monika-soam put the deploy link so the website can check it.

Comment (monika-soam) [reply index.html:100]: @keroloussamy , It's fixed now and I have checked on website too, thank you for your help.

**Review by thecodechaser — APPROVED** [review]
## Hi @monika-soam and Team👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**